detox/ios/Detox/DetoxCrashHandler.mm

```diff
@@ -0,0 +1,153 @@
+//
+//  DetoxCrashHandler.m
```

Comment:
Probably You forgot about one `m` in extension 😄

Reply:
It was an Objective C file before, but now Objective C++.